src/app/admin-manage/admin-manage.controller.js

```diff
@@ -40,32 +40,34 @@
         }
 
         function openCancel(election) {
-
+            $scope.electionArchived = true;
+            $scope.electionType = 'dul';
             var modalInstance = $modal.open({
                 animation: false,
                 templateUrl: 'app/modals/cancel-modal.html',
                 controller: 'Modal',
                 controllerAs: 'Modal',
+                scope: $scope,                                
                 resolve: {
                     election: function () {
                         vm.selectedElection = election;
                     }
                 }
             });
 
-            modalInstance.result.then(function () {
+            modalInstance.result.then(function ($scope) {
```

Comment:
Would it make sense to pass just the value of `$scope.electionArchived` here instead of the whole scope?

Reply:
Modal Controller is being used for many html like cancel, create, confirmation, etc, to receive only electionArchived I have to send it from Modal when I close it, but it will be specific only for cancel, this is why I'm sending scope, but if you want we can create an specific controller to handle this cancel-modal.

Reply:
No worries - I was thinking that the anonymous function could be simplified, but I don't think it can after testing it out. The `$scope` needs to localized there to work correctly.